towhee/runtime/nodes/_flat_map.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 Zilliz. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import List, Any, Generator
+
+
+from .node import Node
+
+
+class FlatMap(Node):
+    """
+    FlatMap Operator.
+
+    FlatMap transforms elements in one row into zero, one, or more elements, i.e. split elements, unnest iterables.
+    """
+    def _get_from_generator(self, gen, size):
+        ret = [ [] for _ in range(size)]
+        for data in gen:
+            for i in range(size):
+                ret[i].append(data[i])
+        return ret
+
+    def process_step(self) -> List[Any]:
+
+        data = self._in_ques[0].get_dict()
+        if data is None:
+            self._set_finished()
+            return True
+
+        nested_data = [data.get(key) for key in self._node_repr.inputs]
```

Comment:
The data `FlatMap` process is not a nested_data, 
for example:
    flat_map('url', 'image', ops.video_decode.ffmpeg()),  in there, the data is url